### PR TITLE
Add listing URLs

### LINKED ISSUE #28: Support opening recordings in new tabs/windows
Support opening listed recordings in new tabs/windows by e.g. right-click menu, ctrl-clicking, middle-button-clicking, and shift-clicking. Consider simply wrapping each cell's contents in an `A` element.

### PR DESCRIPTION
**Pull request by sabbaka**
Close #28.

It's sort of a trick to not get rid of Listing component. Please, take a look.

### CODE REVIEW

File: pkg/session_recording/recordings.jsx [hunk 1]
```diff
@@ -349,16 +350,22 @@
             cockpit.location.go([recording.id], cockpit.location.options);
         }
 
+        createUrl(recording) {
+            let url = cockpit.location.encode([recording.id], cockpit.location.options);
+            return window.location.origin + "/session_recording#" + url;
```
Comment (spbnick) [inline pkg/session_recording/recordings.jsx:355]: I'm a bit uncomfortable with us hardcoding page location and URL structure here. Couldn't the Cockpit library provide us with the full URL somehow? Could you perhaps ask on #cockpit?

**Review by spbnick — COMMENTED** [review]
This looks good, but I'm concerned about the URL generation.

Also, could you please write "Closes" or "Fixes" instead of "Close" and "Fix" in the commit message. I presume GitHub understands those better.

File: pkg/session_recording/recordings.jsx (hunk 1 above)
Comment (sabbaka) [reply pkg/session_recording/recordings.jsx:355]: Sure, I'll check this, for now I've only this way.

Comment (sabbaka) [reply pkg/session_recording/recordings.jsx:355]: One more thing is in cockpit.js window.location is used anyway and some methods are public ( I've also tried to use Location from cockpit.js ) - I'll look more on this and ask, maybe there is a better way.

**Comment by spbnick** [comment]
I tried it out and if I left-click a recording, it opens another cockpit inside the frame, unfortunately.
Middle button works alright. Should we have links underlined, is this in Cockpit conventions?

**Comment by sabbaka** [comment]
Okay, this one I'll do through discussion with Cockpit team, not sure how their URLs are working or maybe I'm missing something, will figure out.

**Pull request closed without merging**